README.md

```diff
@@ -1,14 +1,33 @@
-# Project
+# Azure Quantum QIO Samples
```

Comment:
You've got nice, descriptive, short H1s. :>)
Should this be Microsoft QIO for the brand? -- that's my understanding of the use of QIO.

Reply:
I'll clarify here as per offline discussions. Thanks for raising this!